### PR TITLE
Add new games to game database, new migrations

### PR DESCRIPTION
**Pull request by jacobchrismarsh**

### CODE REVIEW

File: chess/backend/src/game/migrations/0003_auto_20190606_1941.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2 on 2019-06-06 19:41
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
```
Comment (gaganza) [inline chess/backend/src/game/migrations/0003_auto_20190606_1941.py]: What does this line do?

File: chess/backend/src/move/migrations/0002_auto_20190606_1941.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.2 on 2019-06-06 19:41
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('move', '0001_initial'),
```
Comment (gaganza) [inline chess/backend/src/move/migrations/0002_auto_20190606_1941.py]: Same?

File: chess/backend/src/game/migrations/0003_auto_20190606_1941.py (hunk 1 above)
Comment (jacobchrismarsh) [reply chess/backend/src/game/migrations/0003_auto_20190606_1941.py]: I don't know this was auto-generated by `python3 manage.py makemigrations`

File: chess/backend/src/move/migrations/0002_auto_20190606_1941.py (hunk 2 above)
Comment (jacobchrismarsh) [reply chess/backend/src/move/migrations/0002_auto_20190606_1941.py]: auto-generated

**Pull request merged**